### DISCUSSION TITLE
Plesk Obsidian

### DISCUSSION THREAD

**Discussion by JemmaSinclair** [requarks/wiki#6691] [Help / Questions]
Hello!  I am hosting several websites on Plesk Obsidian and would like to host a Wiki.js site on there as well!  
Does anyone have a guide or can provide one for accomplishing this task?

**Comment by 8ternity** [comment]
I found this thread by sercing about the same. Plesk Obsidian seems to not be fully compatible with Composer or Node.JS. It's shared to all the sites hosted in Plesk, from what i've read. I've tried myself today and get same results. 

Like the full permission owner are the main administrator on the main Plesk Root configurations. That's what the hosting provider has telled me.

**Reply by JemmaSinclair** [reply]
In the two years since this I've found that its easier to just host it separately from plesk and use nginx to direct to the proper domain.
